### ISSUE TITLE
[Feature Request] Add an ability to change author on commit

### ISSUE THREAD

**Issue by Lonli-Lokli** [ForkIssues/TrackerWin#1187]
Add an ability to change author on commit, eg in Commit options

**Comment by DanPristupov** [comment]
You can change the identity in the repository settings (Repository -> Settings for this repository).

**Comment by Lonli-Lokli** [comment]
Yep, I know it. But sometimes this feature ('commit as') is very useful

**Comment by Lonli-Lokli** [comment]
I would also add that it might be useful to show identity in multi-workspace setup, to avoid situations when you commit under wrong name
